lib/presenter/Chart.js

```diff
@@ -185,9 +185,8 @@ class Chart extends Events {
         }, this);
 
         if (this.guideLayer) {
-            if (!this._guideLayerEl) {
-                this._guideLayerEl = this.d3Svg.append('g').attr('class', 'guide_layer');
-            }
+            this.d3Svg.select('.guide_layer').remove();
+            this._guideLayerEl = this.d3Svg.append('g').attr('class', 'guide_layer');
```

Comment:
Without this, when graph data updates, the guides show up below the axis text labels leading to ugliness